pkg/webui/console/containers/collaborators-table/index.js

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { Component } from 'react'
+import { defineMessages } from 'react-intl'
+
+import FetchTable from '../fetch-table'
+import Message from '../../../lib/components/message'
+import sharedMessages from '../../../lib/shared-messages'
+import Icon from '../../../components/icon'
+
+import style from './collaborators-table.styl'
+
+const m = defineMessages({
+  id: 'User / Organization ID',
+})
+
+const headers = [
+  {
+    name: 'id',
+    displayName: m.id,
+  },
+  {
+    name: 'isUser',
+    displayName: sharedMessages.type,
+    render (isUser) {
+      const icon = isUser ? 'user' : 'organization'
+
+      return (
+        <span>
+          <Icon icon={icon} className={style.collaboratorIcon} />
+          <Message content={isUser ? sharedMessages.user : sharedMessages.organization} />
+        </span>
+      )
+    },
+  },
+  {
+    name: 'rights',
+    displayName: sharedMessages.rights,
+    render (rights) {
+      return <span>{rights.length}</span>
```

Comment:
we have to remove the tags from the api-keys related pages, we cant translate them

Reply:
Anything is better than just a number. We can look into ways to fix i18n for right names later.

Reply:
cc @htdvisser

Reply:
If you manage to show all rights in the table, then it's fine with me to have them as tags. However, realisticly this isn't going to happen. The descriptions of individual rights will be too long for proper display, and for some collaborators (or API keys) there will be too many rights for proper display.

What are you going to do if a collaborator (or API key) has more rights than you can fit? Are you going to show the first 3? Then you might hide the fact that they have access to sensitive data. Or somehow sort them by how sensitive they are? No idea how we would determine that.

For individual rights, let's consider `RIGHT_APPLICATION_SETTINGS_COLLABORATORS` (view and edit application collaborators). Cutting off the `RIGHT_` isn't going to be sufficient, because `APPLICATION_SETTINGS_COLLABORATORS` won't fit either, and somehow shortening it more will make us lose information. The same problem with the descriptions.

I agree that showing the number of rights isn't ideal, but at least it will be a signal to users that they have to look at the details to get a proper overview.